library/std/src/process.rs

```diff
@@ -1904,8 +1904,8 @@ impl FromInner<imp::ExitCode> for ExitCode {
 }
 
 impl Child {
-    /// Forces the child process to exit. If the child has already exited, an [`InvalidInput`]
-    /// error is returned.
+    /// Forces the child process to exit. If the child has already exited, `Ok(())`
+    /// is returned.
```

Comment:
A few lines below there's an example:

```
child.kill().expect("command wasn't running");
```

message is now incorrect: the only reason why `kill` fails is unknown system error.